core/types/validator.go

```diff
@@ -100,3 +102,24 @@ func (set *validatorList) Contains(addr common.Address) bool {
 	validator := set.Get(addr)
 	return validator != nil
 }
+
+func NewDeposit(amount uint64, unixTimestamp int64) *Deposit {
```

Comment:
Aren't not all amounts `big` numbers?

Reply:
The motivation for this one is that for the console cannot display *big.Int. The other solution is to force a deposit to be a multiplier of the ether denomination and the deposit would be showed in ether. Overflow is possible in the current option since the ether multiplier is 1*10^18. Maybe we should go with the other option and this code would stay like this as 1^10*18 ether is not possible - coins in circulation.

Reply:
I've modified the deposits to return the values in ether (default is wei). In the future we can modify setDeposits to take this into account and also the cmdline flag otherwise this issue will be infinite :)